ds_judgements_public_ui/templates/includes/recent_judgments.html

```diff
@@ -2,8 +2,15 @@
   <div class="recent-judgments__container">
     <h1 id="recently-published-judgments" class="recent-judgments__header">Recently published judgments</h1>
     {% include "includes/judgment_listing.html" with judgments=recent_judgments %}
-    <p class="judgment-listing__more-info">
-      <a href="{% url 'search' %}">See all recent judgments</a>
-    </p>
+    <section class="more-info-footer" aria-labelledby="more-info-footer-heading">
+      <div class="container py-2">
+        <a role="button"
+           aria-label="See all recent judgments"
```

Comment:
I don't think this `aria-label` is necessary if it's exactly the same as the button's label (though @jlhdxw may have other thoughts!)

Reply:
Nope we shouldn't have an aria label here. wcag specifically say not to add aria-labels/titles/alt tags if it doesn't help.

Reply:
removed